doc/HardFloat-Verilog.html

```diff
@@ -1501,12 +1501,14 @@ <H3>9.7. Fused Multiply-Add (<CODE>mulAddRecFN</CODE>,
     mulAddRecFN#(parameter expWidth, parameter sigWidth) (
         input [(`floatControlWidth - 1):0] control,
         input [1:0] op,
+        input int_mul,
         input [(expWidth + sigWidth):0] a,
         input [(expWidth + sigWidth):0] b,
         input [(expWidth + sigWidth):0] c,
         input [2:0] roundingMode,
         output [(expWidth + sigWidth):0] out,
-        output [4:0] exceptionFlags
+        output [4:0] exceptionFlags,
+        output [expWidth + sigWidth-1:0] int_mul_res

```

Comment:
Agree on the 3-bit op.  We can also reuse the output port here, with the topmost bit hardcoded to 0

Reply:
We actually want to have a separate output port, I believe, so our integer multiply can have a lower latency than the FP one, since we don't need to do any re-encoding.

Reply:
Oh, I see. We’ll have to make sure the tools can handle retiming with multiple outputs well (they should be able to)